Testscripts/Linux/VERIFY-BOOT-ERROR-WARNINGS.py

```diff
@@ -97,4 +99,16 @@ def RemoveIgnorableMessages(message_list, keywords_xml_node):
         return None
 
 
+def CollectLogs():
+    logfiles = ['/var/log/messages']
+    hostname = os.uname()[1]
+    for logfile in logfiles:
```

Comment:
What do you need for-loop as you set 'logfiles' to a single file. If you intended to capture all messages, then it should be /var/log/messages*.* with different format in the line 107. If you need a single file copy only, please clarify the goal capturing the latest message file.

Reply:
Hi @juhlee-microsoft ,
It's just for extension. If we want to reserve more log files in the future, it will be convenience to add file names here.